fe/src/stores/useSkillStore.ts

```diff
@@ -41,11 +41,12 @@ export const useSkillStore = defineStore('skills', {
       // start the loading animation
       this.isLoadingSkills = true
       this.skills = []
+      console.log('LOAD SKILLS')
```

Comment:
The `console.log()` police is here! 🚨
Put down the log statement and come out with your hands above your head!